scripts/migrations/2024_06_save_existing_orgs_to_mongodb.pl

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/perl -w
+
+# This file is part of Product Opener.
+#
+# Product Opener
+# Copyright (C) 2011-2023 Association Open Food Facts
+# Contact: contact@openfoodfacts.org
+# Address: 21 rue des Iles, 94100 Saint-Maur des Fossés, France
+#
+# Product Opener is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use Modern::Perl '2017';
+use utf8;
+use Storable qw(lock_retrieve);
+use MongoDB;
+use Encode;
+use ProductOpener::Paths qw/%BASE_DIRS ensure_dir_created/;
+
+my $database = "off";
+my $collection = "orgs";
+
+my $orgs_collection = MongoDB::MongoClient->new->get_database($database)->get_collection($collection);
```

Comment:
I think the issue with MongoDB is here: you should use the Data.pm get_orgs_collection() function instead, as inside docker, mongodb is not on 127.0.0.1, but on "mongodb" host:

www-data@fc8ba1298585:/opt/product-opener$ cat lib/ProductOpener/Config2.pm | grep mongo
		$mongodb
		$mongodb_host
		$mongodb_timeout_ms
my $mongodb_url = $ENV{MONGODB_HOST} || "mongodb";
$mongodb_host = "mongodb://$mongodb_url:27017";
$mongodb = $producers_platform ? "off-pro" : "off";
$mongodb_timeout_ms = 50000;    # config option max_time_ms/maxTimeMS
www-data@fc8ba1298585:/opt/product-opener$ env | grep MONGO
LOG_LEVEL_MONGODB=TRACE
MONGODB_HOST=mongodb